src/journal.rs

```diff
@@ -27,6 +28,8 @@ pub fn print(lvl: u32, s: &str) -> c_int {
     send(&[&format!("PRIORITY={}", lvl), &format!("MESSAGE={}", s)])
 }
 
+pub type JournalWait = Option<time::Duration>;
```

Comment:
I'm not quite sure we need a type alias for this. It might be clearer to use the `Option<time::Duration>` directly & document the meaning of `None` for each function. Otherwise, at least documenting the meaning of `None` (wait forever) on this type alias is probably a good idea.

Reply:
OK with me, I'll remove it and add documentation to the only place where it's used.